### ISSUE TITLE
Is this project maintained?

### ISSUE THREAD

**Issue by naoya7076** [casesandberg/react-color#823]
Consider this is a friendly health check. The last release was over a year ago and last merge was a half year ago . I am seeing some Pull Requests sitting without any activity.

**Comment by bartenra** [comment]
Take a look at https://github.com/omgovich/react-colorful

**Comment by naoya7076** [comment]
Do you mean this project is not maintained?
I checked [react-colorful](https://github.com/omgovich/react-colorful) and I didn't find whether this project is maintained or not.

**Comment by sanbornhilland** [comment]
This project has been a great help to a lot of people and I think it still tends to be the most highly recommended project when people search for a React colour picker. Unfortunately the number of issues and PRs piling up are a big liability. What is the most constructive way that we can revive this project and clear out the backlog?

**Comment by TranquilMarmot** [comment]
Yeah, I tried this library out and the latest version is just plain broken. See also https://github.com/casesandberg/react-color/issues/674 (which is closed...?)

**Comment by zacharytyhacz** [comment]
```
 _______________________________
< does anyone have a good fork? >
 -------------------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||

```

**Comment by jukkahuuskonen** [comment]
> Yeah, I tried this library out and the latest version is just plain broken. See also #674 (which is closed...?)

I have no problems using the latest version ([v.2.19.3](https://www.npmjs.com/package/react-color)). Works just fine, haven't run into any bugs yet.

BUT, I'm also wondering if this project is dead? Last publish about a year ago, PRs just hanging, no updates on v3...

**Comment by Xhale1** [comment]
🎉 I just released a new fork, [@hello-pangea/color-picker](https://github.com/hello-pangea/color-picker)!

It should be a drop-in replacement, just change your imports and you should be good to go 👍

**Comment by antonlimar** [comment]
Hi everyone,

Since this issue asks whether a fork would be the practical path forward: I’ve been maintaining a modernization fork here:

https://github.com/antonlimar/react-color

I opened a fuller migration/announcement issue with the details here: https://github.com/casesandberg/react-color/issues/962

The main focus is keeping the original API usable in modern React projects: TypeScript-compatible source and public typings, ESM package output, modern docs, Storybook, Vitest, and better styling hooks.

Huge thanks to the original maintainers and contributors. My intent is not to replace the history of this project, but to provide a maintained continuation path for teams that still depend on the original API.
